msbuild/Xamarin.MacDev.Tasks/Tasks/ReadAppManifestTaskBase.cs

```diff
@@ -70,7 +67,7 @@ public override bool Execute ()
 			if (Platform == ApplePlatform.MacCatalyst) {
 				// The minimum version in the Info.plist is the macOS version. However, the rest of our tooling
 				// expects the iOS version, so expose that.
-				if (!MacCatalystSupport.TryGetiOSVersion (Sdks.GetAppleSdk (Platform).GetSdkPath (SdkVersion, false), MinimumOSVersion!, out var convertedVersion, out var knownMacOSVersions))
+				if (!MacCatalystSupport.TryGetiOSVersion (Sdks.GetAppleSdk (Platform).GetSdkPath (string.Empty, false), MinimumOSVersion!, out var convertedVersion, out var knownMacOSVersions))
```

Comment:
Why not an overload that just takes false, just for the code to read cleaner.

Reply:
I'll do that in a different PR (since it involves another repo).